hw/ip/sysrst_ctrl/doc/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "SysRst_Ctrl--Chrome OS platform reset related functionality"
+---
+
+# Overview
+
+This document specifies the Chrome OS reset and system related functionality.
+This IP block implements keyboard and button combo triggered action and EC+GSC reset logic.
+This module conforms to the [Comportable guideline for peripheral functionality.]({{< relref "doc/rm/comportability_specification" >}}).
+See that document for integration overview within the broader top level system.
+
+## Features
+
+The IP block implements the following features:
+
+- Always on: uses the always-on power and clock domain
+- EC reset pulse duration control and stretching
+- Keyboard and button combo triggered action
+- AC_present can trigger interrupt 
+- Configuration registers can be set and locked until the next chip reset
+- Pin output override
+
+## Description
+
+The sysrst_ctrl logic is very simple. It looks up the configuration registers to decide how long the EC reset pulse duration and how long the keyboard debounce timer should be. Also what actions to take(eg. Interrupt, EC reset, GSC reset, disconnect the battery from the power tree).
```

Comment:
Can you go through this document and reformat it such that there is only one sentence per line? Thanks!

Also: nit: missing space before parenthesis `take(eg. ` -> `take (e.g.`

Reply:
also, I would suggest to consistently use `e.g.` for `eg.` (there are more variations of this further below)

Reply:
Sure. One sentence per line. Use. e.g.